.github/workflows/sync-docs.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       - name: Update snapshot in consolidated repository
         if: ${{ env.TO_BE_CANCELLED == 'false' }}
         run: |
-          cd ${REPO_CONSOLIDATE}
+          cd../ ${REPO_CONSOLIDATE}
```

Comment:
There is for sure a Syntax error here. Please look at the whitespaces

Reply:
Its fixed . Please review it.

Reply:
Its fixed . Please review it.